Update the Swedish translation (Engine)

matsduf opened the pull request:
## Purpose

This PR updates the Swedish translation of the messages and test case descriptions.

## How to test this PR

Review this PR.

matsstralbergiis reviewed (commented):
"Omvandlingen av den versala (stora) bokstaven \"{unicode_name}\" till "
 "motsvarande gemen (lilla) bokstaven är oklar. Använd bara gemener (små "
 "bokstäver) istället."

"Undvik användning av den versala (stora) bokstaven \"{unicode_name}\""
 "då omvandlingen till motsvarande gemen (lilla) bokstav är oklar."
 "Använd bara gemener (små bokstäver) istället."


"Domännamnet har domändelen (\"label\") \"{label}\", som bara består av ASCII-"
 "tecken, men som innehåller ett otillåtet tecken."

"Domännamnet har domändelen (\"label\"), som bara består av ASCII-tecken, "
 "men som innehåller ett otillåtet tecken."


"Namnserverns adress måste vara globalt routbart"
"Namnserverns adress måste vara globalt routbar"


"Bakåtuppslagning (PTR) finns för namnserververns IP-adress"
"Bakåtuppslagning (PTR) finns för namnserverns IP-adress"


"Nakåtuppslagningen (PTR) stämmer med namnservernamnet"
"Bakåtuppslagningen (PTR) stämmer med namnservernamnet"


"Namnserver {ns} ...."
"Namnservern {ns} ...."


"AS-spridning"
Det måste gå att hitta ett bättre ord här.


"Nameserver ..."   flera ställen i översättningen
"Namnservern ..."


"Namnserver {ns} svarar inte på NS-frågor öve TCP."
"Namnservern {ns} svarar inte på NS-frågor över TCP."


"Konsekvent mellan glue-poster och auktoritativa adressposter"
"Samma innehåll i glue-poster och auktoritativa adressposter"


"Konsekvent SOA MNAME"
"Konsekvent användande av SOA MNAME"


"DNSKEY \"RRset\" har inte signerats någon DNSKEY-post som någon DS-post "
"DNSKEY \"RRset\" har inte signerats med någon DNSKEY-post som någon DS-post "


"En SOA-fråga fick ett svar med svaraskoden (RCODE) \"NOERROR\" från {ns}, "
"En SOA-fråga fick ett svar med svarskoden (RCODE) \"NOERROR\" från {ns}, "


"Test med odefiniera EDNS-version"
"Test med odefinierad EDNS-version"


"Den felaktiga svaraskoden (RCODE) \"{rcode}\" i svaret på en fråga med EDNS "
"Den felaktiga svarskoden (RCODE) \"{rcode}\" i svaret på en fråga med EDNS "


"Felaktigt använt @-tecken i SOA RNAME-fältet"
"Otillåten användning av @-tecken i SOA RNAME-fältet"


"Namnservern \"{nsname}\" angiven SOA MNAME slås upp till IP-adress för "
"Namnservern \"{nsname}\" angiven i SOA MNAME slås upp till IP-adress för "


"Namnservernamnet i SOA MNAME är \".\", vilket normal betyder \"ingen server"
"Namnservernamnet i SOA MNAME är \".\", vilket normalt betyder \"ingen server"


"Zonen har en \"Null MX\" med ett preferensvärde som är icke-noll."
"Zonen har en \"Null MX\" med en prioritet som är icke-noll."

matsduf commented:
> "Omvandlingen av den versala (stora) bokstaven "{unicode_name}" till " "motsvarande gemen (lilla) bokstaven är oklar. Använd bara gemener (små " "bokstäver) istället."
> 
> "Undvik användning av den versala (stora) bokstaven "{unicode_name}"" "då omvandlingen till motsvarande gemen (lilla) bokstav är oklar." "Använd bara gemener (små bokstäver) istället."

Updated to

"Domännamnet innehåller den versala (stora) bokstaven \"{unicode_name}\". Det är oklart hur den ska
omvandlas till den motsvarande gemen (lilla) bokstaven. Använd bara gemener (små bokstäver) istället."

> "Domännamnet har domändelen ("label") "{label}", som bara består av ASCII-" "tecken, men som innehåller ett otillåtet tecken."
> 
> "Domännamnet har domändelen ("label"), som bara består av ASCII-tecken, " "men som innehåller ett otillåtet tecken."

Ignored. The parameters should be there.

> "Namnserverns adress måste vara globalt routbart" "Namnserverns adress måste vara globalt routbar"

Fixed.

> "Bakåtuppslagning (PTR) finns för namnserververns IP-adress" "Bakåtuppslagning (PTR) finns för namnserverns IP-adress"

Fixed.

> "Nakåtuppslagningen (PTR) stämmer med namnservernamnet" "Bakåtuppslagningen (PTR) stämmer med namnservernamnet"

Fixed.

> "Namnserver {ns} ...." "Namnservern {ns} ...."

Fixed in multiple places to make it consistent.

> "AS-spridning" Det måste gå att hitta ett bättre ord här.

"AS-redundans"

> "Nameserver ..." flera ställen i översättningen "Namnservern ..."

Fixed in several places.

> "Namnserver {ns} svarar inte på NS-frågor öve TCP." "Namnservern {ns} svarar inte på NS-frågor över TCP."

Fixed.

> "Konsekvent mellan glue-poster och auktoritativa adressposter" "Samma innehåll i glue-poster och auktoritativa adressposter"

"Glue-poster återfinns som auktoritativa adressposter"

> "Konsekvent SOA MNAME" "Konsekvent användande av SOA MNAME"

"Samma SOA MNAME överallt"

> "DNSKEY "RRset" har inte signerats någon DNSKEY-post som någon DS-post " "DNSKEY "RRset" har inte signerats med någon DNSKEY-post som någon DS-post "

Fixed.

> "En SOA-fråga fick ett svar med svaraskoden (RCODE) "NOERROR" från {ns}, " "En SOA-fråga fick ett svar med svarskoden (RCODE) "NOERROR" från {ns}, "

Fixed.

> "Test med odefiniera EDNS-version" "Test med odefinierad EDNS-version"
> 
> "Den felaktiga svaraskoden (RCODE) "{rcode}" i svaret på en fråga med EDNS " "Den felaktiga svarskoden (RCODE) "{rcode}" i svaret på en fråga med EDNS "

Fixad.

> "Felaktigt använt @-tecken i SOA RNAME-fältet" "Otillåten användning av @-tecken i SOA RNAME-fältet"

Fixed.

> "Namnservern "{nsname}" angiven SOA MNAME slås upp till IP-adress för " "Namnservern "{nsname}" angiven i SOA MNAME slås upp till IP-adress för "

Fixed.


> "Namnservernamnet i SOA MNAME är ".", vilket normal betyder "ingen server" "Namnservernamnet i SOA MNAME är ".", vilket normalt betyder "ingen server"

Fixed.

> "Zonen har en "Null MX" med ett preferensvärde som är icke-noll." "Zonen har en "Null MX" med en prioritet som är icke-noll."

Fixed.
